src/sites/doc/docs/react/intro-react.md

```
@@ -0,0 +1,42 @@
+- 🍭 单元测试覆盖率超过 80%，保障稳定性
+- 📖 提供 Sketch 设计资源
+
+## 扫码体验
+
+<img src="https://storage.360buyimg.com/imgtools/cc08cb2760-d70f19c0-0a95-11ee-808d-b1254110adba.png" width="200" alt="NutUI-React" />
+
+## 兼容情况
+
+| 版本 | 支持的框架 | 视觉规范 | 兼容 |
+| --- | --- | --- | --- |
+| @nutui/nutui-react <img src="https://img.shields.io/npm/v/@nutui/nutui-react" /> | React 17\18 | 京东 [APP 10.0](#/resource) 规范 | 现代浏览器以 Chrome >= 51、iOS >= 10.0 |
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)